src/cdk/testing/webdriver/webdriver-element.ts

```diff
@@ -0,0 +1,263 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+  _getTextWithExcludedElements,
+  ElementDimensions,
+  EventData,
+  ModifierKeys,
+  TestElement,
+  TestKey,
+  TextOptions
+} from '@angular/cdk/testing';
+import {Button, By, Key, WebElement} from 'selenium-webdriver';
+
+/**
+ * Maps the `TestKey` constants to WebDriver's `Key` constants.
+ * See https://github.com/SeleniumHQ/selenium/blob/trunk/javascript/webdriver/key.js#L29
+ */
+const keyMap = {
```

Comment:
Would it make sense to put the keyboard stuff in its own file? (the `keyMap` and `toWebDriverModifierKeys`)